lib/job_queue/job_node.c

```diff
@@ -271,66 +272,65 @@ job_queue_node_type * job_queue_node_alloc( const char * job_name ,
                                             job_callback_ftype * exit_callback,
                                             void * callback_arg) {
 
-  if (util_is_directory( run_path )) {
-    job_queue_node_type * node = util_malloc(sizeof * node );
-    node->progress_timestamp = time(NULL);
-    UTIL_TYPE_ID_INIT( node , JOB_QUEUE_NODE_TYPE_ID );
-    {
-      /* The data initialized in this block should *NEVER* change. */
-      node->job_name       = util_alloc_string_copy( job_name );
-
-      if (util_is_abs_path(run_path))
-        node->run_path = util_alloc_string_copy( run_path );
-      else
-        node->run_path = util_alloc_realpath( run_path );
-
-      node->run_cmd        = util_alloc_string_copy( run_cmd );
-      node->argc           = argc;
-      node->argv           = util_alloc_stringlist_copy( argv , argc );
-      node->num_cpu        = num_cpu;
-
-      if (ok_file)
-        node->ok_file = util_alloc_filename(node->run_path , ok_file , NULL);
-      else
-        node->ok_file = NULL;
-
-      if (status_file)
-        node->status_file = util_alloc_filename(node->run_path , status_file , NULL);
-      else
-        node->status_file = NULL;
-      node->confirmed_running = false;
-
-      if (exit_file)
-        node->exit_file = util_alloc_filename(node->run_path , exit_file , NULL);
-      else
-        node->exit_file = NULL;
-
-      node->exit_callback  = exit_callback;
-      node->retry_callback = retry_callback;
-      node->done_callback  = done_callback;
-      node->callback_arg   = callback_arg;
-    }
-    {
-      node->error_reason   = NULL;
-      node->stderr_capture = NULL;
-      node->stderr_file    = NULL;
-      node->failed_job     = NULL;
-    }
-    {
-      node->job_status     = JOB_QUEUE_NOT_ACTIVE;
-      node->queue_index    = INVALID_QUEUE_INDEX;
-      node->submit_attempt = 0;
-      node->job_data       = NULL;                                    /* The allocation is run in single thread mode - we assume. */
-      node->sim_start      = 0;
-      node->sim_end        = 0;
-      node->submit_time    = time( NULL );
-      node->max_confirm_wait= 60*2; /* 2 minutes before we consider job dead. */
-    }
-
-    pthread_mutex_init( &node->data_mutex , NULL );
-    return node;
-  } else
+  if (!util_is_directory( run_path ))
     return NULL;
+
+  res_log_fdebug("alloc job node for %s", job_name);
```

Comment:
This is so much 'normal operation' that I would not say it warrants a message at all.